src/components/Account/index.jsx

```diff
@@ -34,7 +34,10 @@ class Account extends React.Component {
 
 		const { account: prevAccount } = prevProps;
 		const { account } = this.props;
-		if (!prevAccount.get('history').equals(account.get('history'))) {
+
+		const prevAccountHistory = prevAccount.get('history');
+		const accountHistory = account.get('history');
+		if (prevAccountHistory.toJS().length !== accountHistory.toJS().length) {
```

Comment:
instead of `.toJS().length` better use `.size`

Reply:
done